Pod/Classes/PagingMenuController.swift

```diff
@@ -321,8 +323,14 @@ extension PagingMenuController: UIScrollViewDelegate {
         
         move(toPage: nextPage)
     }
-    
+
+    public func scrollViewWillBeginDragging(_ scrollView: UIScrollView) {
+        onMove?(.didScrollStart)
+    }
+
     public func scrollViewDidEndDragging(_ scrollView: UIScrollView, willDecelerate decelerate: Bool) {
+        onMove?(.didScrollEnd)
```

Comment:
`scrollViewDidEndDecelerating` is appropriate for `didScrollEnd `. Same reason as above.